app/form/Mappings.scala

```diff
@@ -36,6 +36,9 @@ import scala.util.matching.Regex
 
 object Mappings extends DateMappings {
 
+  val businessName: Mapping[String] =
```

Comment:
Does this still check that text is non empty? & can we add tests for this error please? Apologies if I missed this & its already done, I couldn't see it.